app/views/categories/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2>New Category</h2>
+<%= form_with model: @category, url: categories_path, method: :post  do |form| %>
+  <div>
+    <%= form.label :name %><br>
+    <%= form.text_field :name, name: 'category[name]' %>
+  </div>
+  <div>
+    <%= form.label :icon %><br>
+    <%= form.text_field :icon, name: 'category[icon]' %>
+  </div>
```

Comment:
- [ ] [REQUIRED] You have done a great job so far @toyman640 , however, there is an issue with the category icon that needs to be resolved. Please, resize the icon as it appears too big and blocks the user from seeing other categories added and it also blocks the successfully added transactions for each category. This will enable a user friendly app and it will align with the project requirement once resolved.

<img width="937" alt="Screenshot 2023-09-30 at 12 56 06" src="https://github.com/toyman640/budget-app/assets/58692590/375793f1-aba4-4a00-9159-6829e46a6638">